cmd/ucpd/ucp-dev.yaml

```diff
@@ -9,8 +9,14 @@
 # - Talk to Portable Resources' Providers on port 8081
 # - Disables metrics and profiler
 #
-location: 'global'
-storageProvider:
+environment:
```

Comment:
Updates to this configuration file make it consistent with the other services. UCP had a different structure in a lot of places.